pkg/validation/internal/annotations.go

```diff
@@ -30,13 +30,13 @@ which are known to be case sensitive. It also checks to see if the olm.propertie
 annotation is defined in order to add a warning if present. This function can be
 used anywhere annotations need to be checked for case sensitivity.
 
-Arguments
+# Arguments
```

Comment:
Could you explain these changes?

Reply:
This has to do with gofmt updates related to bumping the Go version to Go 1.19.

If you take a look at the Headings section of this documentation https://tip.golang.org/doc/comment you can see that as of Go 1.19 the new heading format is similar to Markdown and uses the # prefix:

>    The # syntax was added in Go 1.19. Before Go 1.19, headings were identified implicitly by single-line paragraphs satisfying certain conditions, most notably the lack of any terminating punctuation.
>
>    Gofmt reformats [lines treated as implicit headings](https://github.com/golang/proposal/blob/master/design/51082-godocfmt.md#headings) by earlier versions of Go to use # headings instead. If the reformatting is not appropriate—that is, if the line was not meant to be a heading—the easiest way to make it a paragraph is to introduce terminating punctuation such as a period or colon, or to break it into two lines.